### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by iyerkrithika21**
1. Changes the instructions in ReadMe.txt
2. Added utils folder to the path to enable import of data loader
3. Converted the image to numpy array in data loader __get_item__() 
4. Changed device to "cpu" and added a comment for "gpu"

### CODE REVIEW

File: Project2_Files/python-Scipts/StarterCode/ReadMe.txt [hunk 1]
```diff
@@ -3,14 +3,22 @@ NOTE:- YOU HAVE TO COMPLETE THE CODE INSIDE THE TAGs"Start of your code" and "En
 
 Steps to Complete the Assignment:-
 
-Part 1:- Creating a Noisy DataSet for Part2
+Make sure you have the following python packages:
```
Comment (jadie1) [inline Project2_Files/python-Scipts/StarterCode/ReadMe.txt:6]: PIL and Numpy are installed with Skimage and I don't think we are using opencv-python anywhere, cv2 is being imported in utils.py but not used, so I think we can remove the import 
So we can shorten this to:
1. skimage
2. matplotlib
3. pandas
4. pytorch

Comment (iyerkrithika21) [reply Project2_Files/python-Scipts/StarterCode/ReadMe.txt:6]: In order to install cv2 you need to run `pip install opencv-python`
https://pypi.org/project/opencv-python/

Comment (jadie1) [reply Project2_Files/python-Scipts/StarterCode/ReadMe.txt:6]: Yeah but we don't need cv2, its an unused import

Comment (tushaarkataria) [reply Project2_Files/python-Scipts/StarterCode/ReadMe.txt:6]: Updated with both of your comments. Yeah, we don't need CV2, it's not being used in the code.

Comment (iyerkrithika21) [reply Project2_Files/python-Scipts/StarterCode/ReadMe.txt:6]: Ohh missed that part.
Thanks
